src/picongpu/include/plugins/IsaacPlugin.hpp

```diff
@@ -400,12 +403,10 @@ class IsaacPlugin : public ILightweightPlugin
                 json_object_set_new( visualization->getJsonMetaRoot(), "cell count", json_string( "Total numbers of cells" ) );
                 json_object_set_new( visualization->getJsonMetaRoot(), "particle count", json_string( "Total numbers of particles" ) );
             }
-            if (visualization->init() != 0)
+            if (visualization->init(reconnect?RetryEverySend:ReturnAtError) != 0)
```

Comment:
This is not outsmarting, this is just plain C (not even C++!) and imo a smart, short solution.
Everone looking at this line of code while not understanding it, should stop looking at PIConGPU code at all. Seriously! And I am not even takling about template meta programming here...

About the release point: Yes, you are probably right. I wanted to wait until the new features are a bit more tested, but I guess for bugs we have the patch version. ;)